integration/k8s_fixture_test.go

```diff
@@ -154,6 +171,9 @@ func (f *k8sFixture) ClearResource(name string) {
 func (f *k8sFixture) CreateNamespaceIfNecessary() {
 	outWriter := bytes.NewBuffer(nil)
 	cmd := exec.CommandContext(f.ctx, "kubectl", "apply", "-f", "namespace.yaml")
+	if f.usingOverriddenConfig {
```

Comment:
instead of having to remember this every time you make a new `Command` object, consider an `f.CommandContext(...)` method that does this automatically (and maybe even passes the ctx for you). Otherwise, seems like a recipe of someone invoking a command wrong and breaking stuff in mysterious ways